src/__tests__/unit/models/time-sync.test.ts

```diff
@@ -909,6 +923,7 @@ describe('execute(): ', () => {
       'start-time': '2023-12-12T00:00:00.000Z',
       'end-time': '2023-12-12T00:00:09.000Z',
       interval: 5,
+      'error-on-padding': false,
```

Comment:
for fully covering the changes, it will be good to add one more test case where panic is enabled, and check if it's doing well

Reply:
absolutely. Done.